internal/pkg/build/files/copy.go

```diff
@@ -17,21 +17,40 @@ import (
 // checks dst and creates parent directories if they do not exist
 // before calling cp
 func Copy(src, dst string) error {
-	_, err := os.Stat(dst)
+	// resolve any bash globbing in filepath
+	paths, err := expandPath(src)
+	if err != nil {
+		return fmt.Errorf("while expanding source path with bash: %s: %s", src, err)
+	}
+
+	_, err = os.Stat(dst)
 	if os.IsNotExist(err) {
```

Comment:
Is dst always going to be a directory? If dst is a file, os.Stat(dst) would return nil but copy can fail.

Reply:
No, dst can be a file. Can you describe the scenario where copy will fail if this is a file?

Reply:
cp /dir/* file

Reply:
I think this a reasonable failure as normally using `cp` in this manner should fail and I don't see a reason to do something like remove the file if it does exist and create a directory since it's likely that isn't the intention of the user

Reply:
Agreed. As long as we send a failure it's fine. It's not right to delete the file and create a directory anyway.